iac/compositions/cat-full/ecs_cluster.tf

```diff
@@ -15,15 +15,15 @@ module "ecs_cluster" {
 }
 
 resource "aws_iam_role" "ecs_execution_role" {
-  name        = "ecs-execution"
+  name        = "${var.resource_name_prefixes.hyphens_lower}-ecs-execution"
```

Comment:
:) I actually chose singularity names for roles and permissions as a signal that we were doing one installation per AWS account. Plus I really like succinct names for IAM entities.

But YES OK if we must, let's namespace them.